addons/website_customer/models/res_partner.py

```diff
@@ -22,12 +22,12 @@ class Tags(models.Model):
 
     @api.model
     def get_selection_class(self):
-        classname = ['default', 'primary', 'success', 'warning', 'danger']
+        classname = ['light', 'primary', 'success', 'warning', 'danger']
```

Comment:
This requires a migration script then, to update existing tags using "default" as selection value.

Reply:
Hello, thanks for the review ! I made [this upgrade PR](https://github.com/odoo/upgrade/pull/4690) but i'm not really confident with upgrades and therefore not exactly sure of what I'm doing/have to do (ex: filename, version to target etc)... Can you maybe have a quick look ?